.github/workflows/ci.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: CI
+
+on: [push, pull_request]
```

Comment:
Just `pull_request` by itself will re-run the action when someone pushes to a branch with an open PR, so no need for `push` unless you intended for it to run when branches that don't have an open PR are pushed to

Reply:
@cdauphinee Yes; that was my original intention. However, I've pushed another change making it the following, which ensures we don't invoke redundant CI jobs when pushing to a branch for which there's an open PR.

```yaml
on:
  pull_request:
  push:
    branches:
      - master
```

Does that address your concern?

Reply:
Got it, that makes sense!